services/getFaceCoordinate.js

```diff
@@ -0,0 +1,34 @@
+const { createCanvas, loadImage } = require("canvas");
+const blazeface = require("@tensorflow-models/blazeface");
+const tf = require("@tensorflow/tfjs-node");
+const fs = require("fs").promises;
+
+function getFaceCoordinate(imgPath) {
+  // eslint-disable-next-line no-async-promise-executor
+  return new Promise(async (resolve, reject) => {
+    try {
+      const imgBuffer = await fs.readFile(imgPath);
+      const img = await loadImage(imgBuffer);
+      const canvas = createCanvas(img.width, img.height);
+      const ctx = canvas.getContext("2d");
+
+      ctx.drawImage(img, 0, 0, img.width, img.height);
+
+      const model = await blazeface.load();
+      const input = tf.browser.fromPixels(canvas);
+      const predictions = await model.estimateFaces(input, false);
+      const faceCoordinates = [];
```

Comment:
여기 선언된 변수명들이 대체로 general해서 상세하게 파악하기 힘든데 이게 원래 컨벤션일까요? 만약 아니라면 대략적으로 알 수 있게 변경해주시면 추후에 작업할때 도움이 많이 될 것 같습니다!
예를 들어,
```suggestion
      const faceImgBuffer = await fs.readFile(imgPath);
      const img = await loadImage(imgBuffer);
      const canvas = createCanvas(img.width, img.height);
      const ctx = canvas.getContext("2d");

      ctx.drawImage(img, 0, 0, img.width, img.height);

      const model = await blazeface.load();
      const input = tf.browser.fromPixels(canvas);
      const predictions = await model.estimateFaces(input, false);
      const faceCoordinates = [];
```

Reply:
해당 함수는 util 함수에서 처리해주고 있어 삭제했습니다. 좋은 피드백 감사합니다 민석님!